develop/tutorials/articles/100-tooling/04-liferay-workspace/05-managing-the-target-platform-for-liferay-workspace.markdown

```diff
@@ -0,0 +1,153 @@
+target. For example,
+
+    liferay.workspace.target.platform.version=7.1.0-m1
+
+<!-- TODO: Update above version to accurate milestone version syntax. This was a
+shot in the dark. -Cody -->
+
+Once the target platform is configured, check to make sure no dependencies in
+your Gradle build files specify a version. The versions are now imported from
+the configured target platform's BOM. For example, a simple MVC portlet's
+`build.gradle` may look something like this:
+
+    dependencies {
+        compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel"
+        compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib"
+        compileOnly group: "javax.portlet", name: "portlet-api"
+        compileOnly group: "javax.servlet", name: "javax.servlet-api"
+        compileOnly group: "jstl", name: "jstl"
+        compileOnly group: "org.osgi", name: "osgi.cmpn"
+    }
+
++$$$
+
+**Note**: The `liferay.workspace.target.platform.version` property also sets
+the distro JAR used in workspace to validate your projects during the build
+process. See the
+[Validating Modules Against the Target Platform](/develop/tutorials/-/knowledge_base/7-1/validating-projects-against-the-target-platform)
+tutorial for more info.
+
+$$$
+
+The target platform functionality is available in Liferay Workspace version
+1.8.1+. If you have an older version, you must update it to leverage platform
+targeting. See the
+[Updating Liferay Workspace](/develop/tutorials/-/knowledge_base/7-1/updating-liferay-workspace)
+tutorial to do this.
+
+<!-- TODO: Verify that the 1.8.1 version is accurate once the features are
+officially released. -Cody -->
+
+You now know how to configure a target platform in workspace and how
+dependencies without versions appear in your Gradle build files. You're all set!
+
+## Targeting a Platform Outside of Workspace
+
+<!-- TODO: Remove this section once the reference doc for the Gradle plugin is
+available. This is just gives a taste of targeting a platform for those who
+don't want to use Workspace. More details will be provided in reference doc.
+-Cody -->
+
+If you prefer to not use Liferay Workspace, but still want to target a platform
+to develop against, you must apply the
+[Target Platform](https://github.com/liferay/liferay-portal/tree/master/modules/sdk/gradle-plugins-target-platform)
+Gradle plugin to the root `build.gradle` file of your custom multi-module Gradle
+build.
+
+To do this, your `build.gradle` file should look similar to this:
+
+    buildscript {
+        dependencies {
+            classpath group: "com.liferay", name: "com.lifereay.gradle.plugins.target.platform", version "1.0.0"
+        }
+        repositories {
+            maven {
+                url "https://cdn.lfrs.sl/repository.liferay.com/nexus/content/groups/public"
+            }
+            maven {
+                url "https://repository.lifray.com/nexus/service/local/repositories/liferay-public-snapshots/content"
```

Comment:
This won't be needed once we officially release the BOMs on our public repo

Reply:
@gamerson this will still be required for the current milestone releases since the portal artifacts weren't released, right? Or is it unnecessary for m1?